fy22_cecil_b_moore/QAQC_mapuser_table.sql

```diff
@@ -0,0 +1,14 @@
+ALTER TABLE pins_mapuser 
+ADD COLUMN q5_gender varchar;
+
+UPDATE pins_mapuser SET q5_gender = 'Male' WHERE q5 = 'Male';
+
+UPDATE pins_mapuser SET q5_gender = 'Female' WHERE q5 = 'Female';
+
+UPDATE pins_mapuser SET q5_gender = 'Male' WHERE q5 = 'male';
+
+UPDATE pins_mapuser SET q5_gender = 'Female' WHERE q5 = 'female';
+
+UPDATE pins_mapuser SET q5_gender = 'Male' WHERE q5 = 'cisgender male';
+
+UPDATE pins_mapuser SET q5_gender = 'Female' WHERE q5 = 'F';
```

Comment:
this works fine,  but you could also do this in a less verbose manner and get rid of the multiple `UPDATE` commands with a single `CASE` statement. e.g.:

```
 UPDATE pins_mapuser SET q5_gender = 
    CASE WHEN lower(q5) in ('female', 'f') then 'Female'
               WHEN lower(q5) in ('male', 'cisgender male') then 'Male' END;
```